packages/contracts-bedrock/src/L1/OptimismPortal2.sol

```diff
@@ -625,7 +625,30 @@ contract OptimismPortal2 is Initializable, ResourceMetering, ISemver {
                 uint256(0), // value
                 uint64(SYSTEM_DEPOSIT_GAS_LIMIT), // gasLimit
                 false, // isCreation,
-                abi.encodeCall(IL1Block.setGasPayingToken, (_token, _decimals, _name, _symbol))
+                abi.encodeCall(IL1Block.setConfig, (_type, _value))
+            )
+        );
+    }
+
+    /// @notice Calls the L2ProxyAdmin as the DEPOSITOR_ACCOUNT. This function can be used
+    ///         to upgrade the predeploys on L2. Only callable by the upgrader role on the
+    ///         SuperchainConfig.
+    function upgrade(uint32 _gasLimit, bytes memory _calldata) external {
```

Comment:
consider changing to u64.

Reply:
Now is the time to change this, I think its fine either way

Reply:
Ok no modification to this